File: challenges/1-produto-simples/go/jessicamosouza/produtoSimples.go
package main

import (
	"fmt"
)

func main() {
	var num1, num2 int

	fmt.Print("Primeiro número: ")
	fmt.Scan(&num1)

	fmt.Print("Segundo número: ")
	fmt.Scan(&num2)

	prod := num1 * num2

	fmt.Printf("%v * %v = %v \n", num1, num2, prod)
}

File: challenges/1-produto-simples/C/daycarol8/produtosimples.c
#include <stdio.h>

int main(void) {

  int num1, num2;
  printf("Digite o primeiro número: ");
  scanf("%d", &num1);

   printf("Digite o segundo número: ");
  scanf("%d", &num2);
  
  int produto = num1 * num2;
  printf("Sua conta: %d x %d = %d",num1,num2,produto);
  return 0;
}